src/Extension/PhiremockProcess.php

```diff
@@ -52,32 +52,20 @@ public function start($ip, $port, $path, $logsPath, $debug, $expectationsPath)
         $phiremockPath = is_file($path) ? $path : $path . DIRECTORY_SEPARATOR . 'phiremock';
         $expectationsPath = is_dir($expectationsPath) ? $expectationsPath : '';
 
-        $this->logPhiremockCommand($ip, $port, $debug, $expectationsPath, $phiremockPath);
         $this->initProcess($ip, $port, $debug, $expectationsPath, $phiremockPath);
+        $this->logPhiremockCommand($debug);
         $logFile = $logsPath . DIRECTORY_SEPARATOR . self::LOG_FILE_NAME;
         $this->process->start(function ($type, $buffer) use ($logFile) {
             file_put_contents($logFile, $buffer, FILE_APPEND);
         });
-        $this->setUpProcessCompatibility();
     }
 
     /**
      * Stops the process.
      */
     public function stop()
     {
-        if ($this->isPcntlEnabled()) {
-            $this->process->signal(SIGTERM);
-            $this->process->stop(3, SIGKILL);
-        }
-    }
-
-    private function setUpProcessCompatibility()
-    {
-        $this->process->setEnhanceSigchildCompatibility(true);
-        if ($this->isWindows()) {
-            $this->process->setEnhanceWindowsCompatibility(true);
-        }
+        $this->process->stop(3);
```

Comment:
It does the same as previous implementation, first it sends SIGTERM and next it sends SIGKILL if process is still running.
```
    public function stop($timeout = 10, $signal = null)
    {
        $timeoutMicro = microtime(true) + $timeout;
        if ($this->isRunning()) {
            // given `SIGTERM` may not be defined and that `proc_terminate` uses the constant value and not the constant itself, we use the same here
            $this->doSignal(15, false);
            do {
                usleep(1000);
            } while ($this->isRunning() && microtime(true) < $timeoutMicro);

            if ($this->isRunning()) {
                // Avoid exception here: process is supposed to be running, but it might have stopped just
                // after this line. In any case, let's silently discard the error, we cannot do anything.
                $this->doSignal($signal ?: 9, false);
            }
        }

        if ($this->isRunning()) {
            if (isset($this->fallbackStatus['pid'])) {
                unset($this->fallbackStatus['pid']);

                return $this->stop(0, $signal);
            }
            $this->close();
        }

        return $this->exitcode;
    }
```